test/lib/oauth.js

```diff
@@ -140,26 +153,29 @@ var authorize = function(cl, rt, user, hostname, port, cb) {
                   pathname: "/oauth/authorize",
                   query: {oauth_token: rt.token}});
 
-    browser.on("error", function(err) {
-        cb(err, null);
-    });
-
     browser.visit(url)
         .then(function() {
             browser
                 .fill("#username", user.nickname)
                 .fill("#password", user.password)
                 .pressButton("#authenticate", function() {
-                    // is there an authorize button?
                     if (browser.button("#authorize")) {
-                        // if so, press it
                         browser.pressButton("#authorize", function() {
-                            cb(null, browser.text("#verifier"));
+                            var res;
+
+                            res = browser.text("#verifier");
+                            browserClose(browser);
+                            cb(null,res); 
```

Comment:
Seems to me like since `res` isn't used anywhere else, the original `cb(null, browser.text("#verifier");` is cleaner. (Maybe I'm being too pedantic, though. Feel free to disagree.)